src/screens/Profile/screens/MyProfile/MyProfile.tsx

```diff
@@ -299,56 +323,89 @@ function MyProfile(props: MyProfileProps) {
                     </>
                   )}
 
-                  {!route.params?.unVerified && !isWeb() ? (
-                    <>
-                      <Button
-                        style={styles.button}
-                        caption={t('offlineMap.title')}
-                        variant="tertiary"
-                        onPress={handleNavigateOfflineMap}
-                      />
-                      <Spacer times={4} />
-                    </>
-                  ) : null}
+                  {/*{!route.params?.unVerified && !isWeb() ? (*/}
+                  {/*  <>*/}
+                  {/*    <Button*/}
+                  {/*      style={styles.button}*/}
+                  {/*      caption={t('offlineMap.title')}*/}
+                  {/*      variant="tertiary"*/}
+                  {/*      onPress={handleNavigateOfflineMap}*/}
+                  {/*    />*/}
+                  {/*    <Spacer times={4} />*/}
+                  {/*  </>*/}
+                  {/*) : null}*/}
 
                   {planterData?.planterType && !!wallet ? (
-                    <Invite style={styles.button} address={wallet} planterType={Number(planterData?.planterType)} />
-                  ) : null}
-
-                  <Button
-                    style={styles.button}
-                    caption={t('settings.title')}
-                    variant="tertiary"
-                    onPress={handleNavigateSettings}
-                  />
-                  <Spacer times={4} />
-                  <Button
-                    style={styles.button}
-                    caption={t('support')}
-                    variant="tertiary"
-                    onPress={handleNavigateSupport}
-                  />
-                  {profile.isVerified ? (
-                    <>
+                    <ProfileGroupButton isVerified={profile.isVerified} direction="row">
```

Comment:
ProfileGroupButton
I've noticed you are rendering this component with the direction='row' prop most of the time.
Make it default value